modules/docs/arrow-docs/docs/docs/quickstart/blogs/README.md

```diff
@@ -97,3 +97,9 @@ Kotlin Brooklyn Meetup, Jan ’18 - version 0.6.1
 * [Validated](http://danielecampogiani.com/blog/2018/02/android-functional-validation-4-validated/)
 
 April ’18 - version 0.7
+
+### 1/n - How do I ... in FP : Validation
+
+[Emmanuel Nhan](https://www.enhan.eu/author/enhan/) showcases different approaches to validation including examples for `ValidatedNel` from Arrow in this great and in depth post using Kafka Streams Config parameters as example.
+
+[1/n - How do I ... in FP : Validation](https://www.enhan.eu/how-to-in-fp/)
```

Comment:
Please add the version number for the article!

Reply:
done